### ISSUE TITLE
Cyrillic language support

### ISSUE THREAD

**Issue by Thomas101** [Wulf/nodehun#34]
Hi,

I've been playing around with nodehun and it doesn't look like it works with cyrillic languages (e.g. greek & russian) in that it always returns that the word is misspelled and there are no suggestions. 

This is the code I was testing with. In both languages I took "Hello" off Google translate, then a random word out of the aff file. I was wondering if this was an encoding issue somewhere?

```js
var nodehun = require('nodehun');
var ruRU = require('dictionary-ru')
var elGR = require('dictionary-el-gr');

ruRU(function (err, result) {
  var dict = new nodehun(result.aff, result.dic);
  dict.spellSuggest('Здравствуйте',function(err, correct, suggestion, origWord){
      console.log('RU', err, correct, suggestion, origWord);
  });

  dict.spellSuggest('Эдуардович',function(err, correct, suggestion, origWord){
      console.log('RU', err, correct, suggestion, origWord);
  });
})

elGR(function (err, result) {
  var dict = new nodehun(result.aff, result.dic);
  dict.spellSuggest('Χαίρετε',function(err, correct, suggestion, origWord){
      console.log('GR', err, correct, suggestion, origWord);
  });

  dict.spellSuggest('αβαθμίδωτες',function(err, correct, suggestion, origWord){
      console.log('GR', err, correct, suggestion, origWord);
  });
})

```

```
RU null false null Здравствуйте
RU null false null Эдуардович
GR null false null Χαίρετε
GR null false null αβαθμίδωτες
```

Thanks for you help!

**Comment by danielbankhead** [comment]
Hey @Thomas101, are the `.aff` & `.dic` files in UTF-8?

You can check the encoding with `enca` (installed via brew, yum, etc.)
- Ex: `enca -gL russian ru_RU.dic`

And convert with `iconv`
- Ex: `iconv -f KOI8-R -t UTF-8 ru_RU.aff >> ru_RU-utf8.aff`


Then try `здравствуйте` instead of `Здравствуйте` for 'Hello' (Google Translate is slightly off).

**Comment by Thomas101** [comment]
Thanks for looking into this! It does look like it's utf-8. Enca reports...

```
MacBook-Pro:dictionary-ru Tom$ enca -gL russian index.dic
Universal transformation format 8 bits; UTF-8
MacBook-Pro:dictionary-ru Tom$ enca -gL russian index.aff
Universal transformation format 8 bits; UTF-8
```

I grabbed the dictionary from [this repo](https://github.com/wooorm/dictionaries/tree/master/dictionaries/ru_RU)

I did spot the `3` vs `з` from google translate. I'm not a russian speaker and wasn't sure what the correct thing to do here was hence why I dragged a second word out the dictionary just in case :-)
